modules/rosa-upgrading-automatic-ocm.adoc

```diff
@@ -21,3 +35,7 @@ The scheduling options are displayed.
 . Click *Save*.
```

Comment:
It's not part of your update, but sharing about "drop-down list"...ISG has it listed as a UI element, but the usage and examples encourage referring to the element as a "list" (omitting "drop-down").   Maybe some other PR.

Reply:
Thanks for highlighting this, it prompted me to review the ISG updates on this topic. I have removed "drop-down" from the sentence.